kickoff/jsonexportl10n.py

```diff
@@ -0,0 +1,94 @@
+import os
+from collections import defaultdict
+from os import path
+import json
+
+from flask import render_template
+
+from kickoff import app
+from kickoff import config
+
+from mozilla.release.l10n import parsePlainL10nChangesets
+
+from kickoff.model import getReleases
+from kickoff.model import getReleaseTable
+from jsonexport import myjsonify
+
+
+@app.route('/json/l10n/<releaseName>.json', methods=['GET'])
+def l10nExport(releaseName):
+    # Export the l10n changeset for a product
+    releaseTable = getReleaseTable(releaseName)
+    release = releaseTable.query.filter_by(name=releaseName).first()
+
+    if release is None or release.l10nChangesets == "legacy":
+        return myjsonify({})
+
+    locale_list = defaultdict()
+    if "Firefox" in releaseName or "Thunderbird" in releaseName:
+        locales = parsePlainL10nChangesets(release.l10nChangesets)
+        for key, changeset in locales.iteritems():
+            locale_list[key] = {
+                "changeset": changeset,
+            }
+
+    if "Fennec" in releaseName:
+        locales = json.loads(release.l10nChangesets)
+        for key, extra in locales.iteritems():
+            locale_list[key] = {
+                "changeset": extra['revision'],
+                "platforms": extra['platforms'],
```

Comment:
Do we need to specify platforms here? I talked to Axel in mozlondon and we want to kill the platforms filed here. We get it from the l10n dashboard, which generates it using an in-tree file https://dxr.mozilla.org/mozilla-central/source/mobile/android/locales/maemo-locales.